asllib/Parser.mly

```diff
@@ -504,7 +504,7 @@ let loop_limit == { None }
 
 let stmt ==
   annotated (
-    | terminated_by(END,
+    | terminated_by(END; SEMI_COLON,
```

Comment:
Please make this rule optional.
When an `end` is not followed by a semicolon, please emit a warning (just once for the entire run, otherwise we'll get thousands of warnings).